cardano-cli/src/Cardano/CLI/Shelley/Run/Key.hs

```diff
@@ -425,6 +448,26 @@ convertITNSigningKey privKey = do
     Just signKey -> Right $ StakeSigningKey signKey
     Nothing -> Left $ SigningKeyDeserializationError keyBS
 
+-- | Convert extended private ed22519 key to a Shelley signing key
+-- Extended private key = 64 bytes,
+-- Public key = 32 bytes.
+convertITNExtendedSigningKey :: Text -> Either ConversionError (SigningKey StakeExtendedKey)
+convertITNExtendedSigningKey privKey = do
+  (_, _, privkeyBS) <- decodeBech32Key privKey
+  let dummyChainCode = "00000000000000000000000000000000"
```

Comment:
```suggestion
  let dummyChainCode = BS.replicate 32 0
```

@dcoutts: Would it matter if these are null bytes?

Reply:
0x0 vs '0' is indeed better, and your suggestion to use replicate makes the length clearer.